plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/BridgingHiveMetastore.java

```diff
@@ -336,7 +337,12 @@ public void commentColumn(String databaseName, String tableName, String columnNa
         io.trino.hive.thrift.metastore.Table table = delegate.getTable(databaseName, tableName)
                 .orElseThrow(() -> new TableNotFoundException(new SchemaTableName(databaseName, tableName)));
 
-        for (FieldSchema fieldSchema : table.getSd().getCols()) {
+        List<FieldSchema> fieldSchemas = ImmutableList.<FieldSchema>builder()
+                .addAll(table.getPartitionKeys())
+                .addAll(table.getSd().getCols())
```

Comment:
nit: I would replace the order as we usually process data columns first. 
```suggestion
                .addAll(table.getSd().getCols())
                .addAll(table.getPartitionKeys())
```